mpt_extension_sdk/constants.py

```diff
@@ -1,7 +1,13 @@
+from enum import Enum
+
 EVENT_TYPES = "orders"
 SECURITY_ALGORITHM = "HS256"
 USER_AGENT = "swo-extensions/1.0"
 CONSUME_EVENTS_HELP_TEXT = "Consume events from the MPT platform"
 DEFAULT_APP_CONFIG_GROUP = "swo.mpt.sdk"
 DEFAULT_APP_CONFIG_NAME = "app_config"
 DJANGO_SETTINGS_MODULE = "mpt_extension_sdk.runtime.djapp.conf.default"
+
+
+class NotifyCategories(Enum):
+    ORDERS = "NTC-0000-0006"
```

Comment:
This thing should a configuration variable... means --> should be passed through env, like tokens, webhooks secrets, product ids, etc

Reply:
Done!

Reply:
Done.